src/nupic/utils/SdrMetrics.cpp

```diff
@@ -117,32 +117,32 @@ std::ostream& operator<<(std::ostream& stream, const Sparsity &S)
 
 /******************************************************************************/
 
-ActivationFrequency::ActivationFrequency( const vector<UInt> dimensions, UInt period )
+ActivationFrequency::ActivationFrequency( const vector<UInt> &dimensions,
+                                          UInt                period,
+                                          Real                initialValue )
```

Comment:
nit, const, const

Reply:
Done.

I just changed a bunch of things in the SDR & Metrics to use `const &` where able and appropriate.